app/views/users/_following.html.erb

```diff
@@ -0,0 +1,34 @@
+<ul class="accordion" data-accordion data-allow-all-closed="true">
+
+  <% @followable_types.each do |followable_type| %>
+
+    <li class="accordion-item" data-accordion-item>
+
+      <a href="#" class="accordion-title">
+        <span class="icon">
+          <i class="icon icon-<%= entity_icon(followable_type) %>"></i>
+        </span>
+        <span class="title">
+          <strong><%= entity_title(followable_type) %></strong>
+        </span>
+      </a>
+
+      <div class="accordion-content" data-tab-content>
+        <table>
+          <tbody>
+            <% @follows.where(followable_type: followable_type).each do |follow| %>
+              <% if followable_type == "Proposal" %>
```

Comment:
After seeing both `entity_title` and `entity_icon` helper methods. I think a third one could be nice to avoid spilling `== "Proposal"` and `== "Budget::Investment"` on a view :D, seems tricky to remember if another entity becomes as well "followable"

So the entire if-else could be replace for something like:
```
<%= render entity_partial(followable_type), entity_partial(followable_type).to_sym => follow.followable %>
```

Being the helper function at `app/helpers/follows_helper.rb`:
```
def entity_partial(title)
  title.parameterize.gsub('-','_')
end
```

What do you think?